docarray/documents/audio.py

```diff
@@ -94,11 +94,11 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[AudioUrl]
-    tensor: Optional[AudioTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[AudioBytes]
-    frame_rate: Optional[int]
+    url: Optional[AudioUrl] = None
```

Comment:
why is this needed?

Reply:
None is need each  time there is Optional.

This does not affect the behavior of pydantic v1 and in pydantic v2 you have to add none to have the same behavior as pydantic v1.


FYI: https://docs.pydantic.dev/latest/migration/#required-optional-and-nullable-fields